AsyncDisplayKit/ASPagerNode.m

```diff
@@ -27,29 +27,34 @@ - (instancetype)init
   
   self = [super initWithCollectionViewLayout:_flowLayout];
   if (self != nil) {
-    self.view.asyncDataSource = self;
-    self.view.asyncDelegate = self;
-    
-    self.view.pagingEnabled = YES;
-    self.view.allowsSelection = NO;
-    self.view.showsVerticalScrollIndicator = NO;
-    self.view.showsHorizontalScrollIndicator = NO;
-    
-    ASRangeTuningParameters tuningParams = { .leadingBufferScreenfuls = 1.0, .trailingBufferScreenfuls = 1.0 };
-    [self setTuningParameters:tuningParams forRangeType:ASLayoutRangeTypePreload];
-    [self setTuningParameters:tuningParams forRangeType:ASLayoutRangeTypeRender];
   }
   return self;
 }
 
-- (void)reloadData
+- (void)didLoad
 {
-  [self.view reloadData];
+  [super didLoad];
+  
+  self.view.asyncDataSource = self;
```

Comment:
Stylistically, but also for performance on the oldest devices, prefer creating a local variable for any properties accessed more than once (but especially more than three times).